lib/improver/tests/set_up_test_cubes.py

```diff
@@ -210,11 +216,22 @@ def set_up_variable_cube(data, name='air_temperature', units='K',
         for coord in include_scalar_coords:
             scalar_coords.append((coord, None))
 
+    # set up attributes
+    cube_attrs = {}
+    if standard_grid_metadata is not None:
+        cube_attrs['mosg__model_configuration'] = standard_grid_metadata
+        cube_attrs['mosg__grid_type'] = 'standard'
+        cube_attrs['mosg__grid_version'] = '1.3.0'
+        cube_attrs['mosg__grid_domain'] = (
+            'uk_extended' if 'uk' in standard_grid_metadata else 'global')
```

Comment:
Could we not set these via GRID_ID_LOOKUP via improver.utilities.cube_metadata

uk_ens would become enukx_standard_v1 instead.

Reply:
I'd actually rather not do it that way as it ties us to the definitions of the old grid metadata (<model>_standard_v1).  I could define a new centralised dictionary if you prefer?

Reply:
I can understand you don't want to use old grid metadata but we also don't want these defined in several different places which make maintenance difficult when things change.

In improver.utilities.metadata could we have a centralised dictionary e.g.

MOSG_GRID_DEFINITION = {'uk_ens': {'mosg__grid_type': 'standard',
                                        'mosg__model_configuration': 'uk_ens',
                                        'mosg__grid_domain': 'uk_extended',
                                        'mosg__grid_version': '1.3.0'},
                          'gl_ens': {'mosg__grid_type': 'standard',
                                       'mosg__model_configuration': 'gl_ens',
                                       'mosg__grid_domain': 'global',
                                       'mosg__grid_version': '1.3.0'},
                  'uk_det': {'mosg__grid_type': 'standard',
                                       'mosg__model_configuration': 'uk_det',
                                       'mosg__grid_domain': 'uk_extended',
                                       'mosg__grid_version': '1.3.0'},
                  'gl_det': {'mosg__grid_type': 'standard',
                                      'mosg__model_configuration': 'gl_det',
                                      'mosg__grid_domain': 'global',
                                      'mosg__grid_version': '1.3.0'},
                  }

Reply:
OK, will sort that now.